pyproject.toml

```diff
@@ -2,6 +2,7 @@
 requires = [
     "colorama",
     "defusedxml",
+    "k2l",
```

Comment:
Just a minor comment, and you don't have to change this, I will take care of it later: The requirements list here is reserved for "global" requirements, i.e. things that the refinery framework (and libraries) require. Requirements that are "local" to individual units can be specified in that unit itself, an example would be [the pyperclip dependency in emit](https://github.com/binref/refinery/blob/edc139b9f1520c2a2dee59430cccdebd18fb5cef/refinery/units/meta/emit.py#L19-L26).